liquidity_pool/src/lib.rs

```diff
@@ -44,4 +45,11 @@ pub trait LiquidityPool:
         });
         self.liquidation_threshold().set(&liquidation_threshold);
     }
+
+    #[only_owner]
+    #[endpoint(setTransferExecGasLimit)]
+    fn set_transfer_exec_gas_limit(&self, gas_limit: u64) -> SCResult<()> {
+        self.transfer_exec_gas_limit().set(&gas_limit);
+        Ok(())
+    }
```

Comment:
SCResult return not needed.
```suggestion
    #[only_owner]
    #[endpoint(setTransferExecGasLimit)]
    fn set_transfer_exec_gas_limit(&self, gas_limit: u64) {
        self.transfer_exec_gas_limit().set(&gas_limit);
    }
```